src/Symfony/Component/Ldap/Entry.php

```diff
@@ -22,9 +22,22 @@ class Entry
     public function __construct(string $dn, array $attributes = array())
     {
         $this->dn = $dn;
+        $this->resetValueIndex($attributes);
         $this->attributes = $attributes;
     }
 
+    /**
+     * Resets the index for array value
+     *
+     * @param array $attributes
+     */
+    public function resetValueIndex($attributes = array())
```

Comment:
This function is broken, as it modifies the array by reference, but the argument is not a reference (so `$attributes` is still unchanged in the constructor)

Reply:
my bad, the array_map should be used here :D.